src/productcatalogservice/server.go

```diff
@@ -288,7 +290,11 @@ func (p *productCatalog) GetProduct(ctx context.Context, req *pb.GetProductReque
 	}
 	if found == nil {
 		return nil, status.Errorf(codes.NotFound, "no product with ID %s", req.Id)
-	}
+    }
+    resp, err := http.Get("http://cloud-ops-sandbox-406964100.wl.r.appspot.com/getRating/66VCHSJNUP")
```

Comment:
the value of the server needs to come from the config - it will be different each time the app is deployed on a different sandbox. You also need to set this value in the config as part of provisioning (i.e. first deploy appEngine app, get its address, inject into config of hipstershop for productcatalogservice, then deploy hipstershop with this config

Reply:
> inject into config of hipstershop for productcatalogservice

I don't think we should be re-writing the kubernetes manifests manually. We should probably add it dynamically as a [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-literal-values). We'd also have to add some detection code here to make sure it was set properly

Reply:
I have removed changes from server.go